mappings/net/minecraft/entity/passive/FrogBrain.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS net/minecraft/unmapped/C_gtwuaauj net/minecraft/entity/passive/FrogBrain
+	FIELD f_bguhkofh MAKING_LOVE_SPEED F
```

Comment:
...really

I mean I can't say I can come up with a better name but still lmao

Reply:
LOVE_SPEED or MAKE_BABY_SPEED wait that also sounds weird LMAOOOOO

Reply:
There's `BREEDING_SPEED` in other classes

Reply:
```suggestion
	FIELD f_bguhkofh BREEDING_SPEED F
```